Support en-US localization

yechs opened the pull request:
Closes #11 

暂且是根据 dc8ebe5 commit 下的 POT file 做的翻译，之后 check 一下文本有没有发生改变

目前只是初翻，还需要一定程度的修改，所以只是 draft PR 并且没有 commit mo 文件

yechs committed:
en-US l10n: 12% done

yechs committed:
en-US l10n: all strings done

- Finish translating all strings into en-US
- Awaiting refinement
- Generated mo file NOT commited!

yechs commented:
（这两个 commit 已经 rebase master 了，不过后续更新之后应该还会再 rebase & force-push）

syfxlin commented:
设置部分的文本由于界面更改发生了一些变化，所以还需要更改部分翻译，新的POT文件已推送。

yechs commented:
OK，我之后去 rebase 568c7ce5

syfxlin commented on languages/en_US.po:
```diff
@@ -0,0 +1,734 @@
+
+#: include/config.class.php:17 include/config_style.php:2
+msgid "Origami主题 - 样式"
+msgstr "Origami Theme—Appearances"
```
这里的`-`的两侧添加空格或许会好一点

yechs replied:
因为我这里用的是 em dash(—)，理论上（按照语法规则）两侧是没有空格的 (Source: [Stack Exchange English | Spaces and em-dash](https://english.stackexchange.com/questions/154996/spaces-and-em-dash/154998#154998)

不过我可以改成 en dash(-) 并在两侧加上空格23333

syfxlin commented on languages/en_US.po:
```diff
@@ -0,0 +1,734 @@
+
+#: include/config.class.php:25 include/config_fun.php:2
+msgid "Origami主题 - 功能"
+msgstr "Origami Theme—Features"
```
这里的`-`的两侧添加空格或许会好一点

syfxlin commented on languages/en_US.po:
```diff
@@ -0,0 +1,734 @@
+
+#: include/config.class.php:33 include/config_about.php:2
+msgid "Origami主题 - 关于"
+msgstr "Origami Theme—About"
```
这里的`-`的两侧添加空格或许会好一点

syfxlin commented on languages/en_US.po:
```diff
@@ -0,0 +1,734 @@
+
+#: include/config.class.php:221 include/config.class.php:234
+msgid "如果填入default则会自动读取用户信息显示"
+msgstr "Enter default to automatically fetch user info for display"
```
这里的`default`改成`"default"`会好一点

yechs replied:
ok

syfxlin reviewed (commented):
除了这点小问题，其他应该都没什么问题

yechs committed:
en-US l10n: fix according to comments by @syfxlin

- Use en-dashes surrounded by spaces rather than em-dashes
- Use double quotes to surround the "default" field

yechs commented:
rebase 到 master branch 了，我去更新 strings

yechs committed:
en-US l10n: Update strings from 568c7ce

yechs committed:
en-US l10n: Translate newly updated strings

syfxlin commented:
刚刚我添加了一个设置项，还有之前不小心漏掉的3个字段，能帮忙更新下翻译文本吗？

yechs commented:
OKOK

yechs committed:
en-US l10n: Update strings from a4d3312

yechs committed:
en-US l10n: Finalizing translations

- Translate strings from a4d3312
- Some refinement work

syfxlin commented:
我测试下，等下就发布新版本ヾ(≧▽≦*)o

yechs commented:
> 
> 
> 我测试下，等下就发布新版本ヾ(≧▽≦*)o

好耶 :tada:
